Documentation/Books/Manual/Upgrading/OSSpecificInfo/Windows.md

```diff
@@ -0,0 +1,36 @@
+**TMP**
```

Comment:
Better use HTML comments so that it won't show in devel previews?
```suggestion
<!-- TODO -->
```

Reply:
nope, will be merged only when finished..